.github/workflows/dashboard.yml

```diff
@@ -0,0 +1,20 @@
+name: Dashboard CI/CD
+on:
+  push:
+    branches:
+    - master
```

Comment:
More of a stylistic discussion, but for yaml files, how do you feel about adding an indent when we go into list mode?

E.g.

```yaml
branches:
  - master
```

I see our k8s configs are done with them in the same line, but I was thinking it'd be nice to standardize one on or the other as having both can be confusing.

Reply:
OK—let's standardize on the non-nested version; you probably write the most yaml out of everyone :).

Reply:
professional yaml developer